GUI.py

```diff
@@ -0,0 +1,297 @@
+        self.headless_int = IntVar()
+        self.headless_checkbox = Checkbutton(
+            app, text="Headless", variable=self.headless_int)
+        self.headless_checkbox.grid(row=15, sticky=W, padx=25)
+
+        # No Quit
+        self.no_quit_int = IntVar()
+        self.no_quit_checkbox = Checkbutton(
+            app, text="Dont Quit After Purchase", variable=self.no_quit_int)
+        self.no_quit_checkbox.grid(row=16, sticky=W, padx=25)
+
+        # No Quit
+        self.purchase_int = IntVar()
+        self.purchase_checkbox = Checkbutton(
+            app, text="Purchase", variable=self.purchase_int)
+        self.purchase_checkbox.grid(row=17, sticky=W, padx=25)
+
+        # Start Script
+        self.btn = Button(app, text='Start', width=12, command=self.test)
+        self.btn.grid(row=18, column=0, sticky=W, padx=25)
+
+        # Disabled items since auto check is enabled
+        self.login_time_label.grid_remove()
+        self.login_time_minutes_dropdown.grid_remove()
+        self.login_time_hours_dropdown.grid_remove()
+        self.release_time_label.grid_remove()
+        self.release_time_dropdown.grid_remove()
+        self.retries_label.grid_remove()
+        self.retries_dropdown.grid_remove()
+        self.page_load_timeout_label.grid_remove()
+        self.page_load_timeout_dropdown.grid_remove()
+
+        app.title('Sneaker Bot')
+        app.rowconfigure(20, {'minsize': 30})
+        app.columnconfigure(20, {'minsize': 30})
+
+    def test(self):
+
+        options = webdriver.FirefoxOptions()
+        if(self.headless_int.get() != 0):
+            options.add_argument("--headless")
+
+        if(self.purchase_int.get() != 0):
+            purchase = True
+        else:
+            purchase = False
+
+        if(self.release_time_int.get() != 1):
+            release_time = self.release_time.get() + "h"
+
+        if(self.page_load_int.get() != 1):
+            page_load_timeout = self.page_load_timeout.get()
+
+        driver = webdriver.Firefox(
+            executable_path="./bin/geckodriver", firefox_options=options, log_path=os.devnull)
+
+        if(self.username_entry.get() != "" and self.password_entry.get() != "" and self.url_entry.get() != "" and self.shoe_size.get() != "" and self.cvv_entry.get() != ""):
+            command = 'py .\experimental.py  --username %s --password %s --url %s --shoe-size %s --driver-type %s --cvv %s' % (
+                self.username_entry.get(), self.password_entry.get(), self.url_entry.get(), self.shoe_size.get(), self.driver.get(), self.cvv_entry.get())
+
+            run(driver=driver, shoe_type="M", username=self.username_entry.get(), password=self.password_entry.get(
+            ), url=self.url_entry.get(), shoe_size=self.shoe_size.get(), cvv=self.cvv_entry.get(), purchase=purchase, release_time=release_time, page_load_timeout=page_load_timeout)
+
+            #  Determines if bot features where disabled.
+            # if(self.login_time_int.get() != 1):
```

Comment:
Any reason why all this is commented out?

Reply:
Yes, its old code that was used to edit the console command options executed by os.command() but that method is no longer being used so I have been using that code as a reference to create the new method which just calls the run function directly with the set options. Totally forgot that part wasn't fully converted, still working on setting all options in the run function.  The comments will be removed in the next commit